napari/components/add_layers_mixin.py

```diff
@@ -682,12 +686,26 @@ def add_vectors(
             D dimensions. An (N1, N2, ..., ND, D) array is interpreted as
             "image-like" data where there is a length D vector of the
             projections at each pixel.
+        properties : dict {str: array (N,)}, DataFrame
+            Properties for each vector. Each property should be an array of length N,
+            where N is the number of vectors.
         edge_width : float
             Width for all vectors in pixels.
         length : float
              Multiplicative factor on projections for length of all vectors.
         edge_color : str
             Edge color of all the vectors.
+        edge_color_cycle : np.ndarray, list, cycle
+            Cycle of colors (provided as RGBA) to map to edge_color if a
+            categorical attribute is used to set face_color.
```

Comment:
maybe better to say `to color the vectors` - it is a little confusing to reference `face_color` here if one isn't familiar with the vispy api - which people shouldn't need to be when reading our model files

Reply:
That makes sense. I will update here and in the other places you flagged.